qt/snapcraft.yaml

```diff
@@ -33,3 +33,8 @@ parts:
       - libqt5svg5 # for loading icon themes which are svg
       - appmenu-qt5
 
+#plugs:
+#  ubuntu-app-runtime:
```

Comment:
I would name that plug "platform" (on both sides) as it provides a platform and we want multiple apps to support it. That way, we can standard the plug and slot name for all platform snap. Making sense?